README.md

```diff
@@ -1,59 +1,40 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello Rails-React
 
-> One paragraph statement about the project.
-
-![screenshot](./app_screenshot.png)
-
-Additional description about the project and its features.
+> This is a simple hello world project meant to teach how to use rails-react in two different projects
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo
-
-[Live Demo Link](https://livedemo.com)
-
+- Ruby on Rails
+- React
+- Redux
+- VScode
+- Git & GitHub
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
-
+- Go to a repository on your local machine where you want to clone this repo
+- run `git clone https://github.com/jmontanholi/hello-rails-back-end.git`
+- Open the repo
+- Run these commands in order
+   - `bundle install`
+   - `rails db:create` (maybe you will need to change the database.yml file)
+   - `rails db:migrate`
+   - `rails server`
+- Go to the react project [here](https://github.com/jmontanholi/hello-react-front-end)
+- run `git clone https://github.com/jmontanholi/hello-react-front-end.git`
+- run `npm start` or `yarn start`
+- Open the server in your browser
 ## Authors
```

Comment:
- [ ] OPTIONAL
Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.